components/cloud_usage_modal/mini_modal.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+import {Modal} from 'react-bootstrap';
+
+import {Limits} from '@mattermost/types/cloud';
+import {CloudUsage} from 'components/common/hooks/useGetUsage';
+
+import WorkspaceLimitsPanel, {Message, messageToElement} from './workspace_limits_panel';
+
+import './mini_modal.scss';
+
+interface Props {
+    limits: Limits;
+    usage: CloudUsage;
+    showIcons?: boolean;
+    title: Message | React.ReactNode;
+    onClose: () => void;
+}
+
+export default function MiniModal(props: Props) {
+    const [show, setShow] = useState(true);
+
+    return (
+        <Modal
```

Comment:
there is a component called GenericModal, it's kind of a wrapper on top of Bootstrap Modal. I have seen it used in other several places, not sure if it make sense to continue using that one.

Reply:
I'll take a look!

Reply:
Yep! It works for these with the caveat that I have to provide the compass theme if they are used on the system console page (which is what I think we want to do in this case).

Reply:
got it!